### PR TITLE
add code-scheme placeholders

### PR DESCRIPTION
**Pull request by IsaackMwenda**
Adapted from KEEP-II Kakuma

### CODE REVIEW

File: .idea/Project-WUSC-LEAP-II.iml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (as2388) [inline .idea/Project-WUSC-LEAP-II.iml:1]: I don't think you meant to include these

File: code_schemes/s02e01.json [hunk 2]
```diff
@@ -0,0 +1,178 @@
+{
+  "SchemeID": "Scheme-eea92621",
+  "Name": "S02E01",
```
Comment (as2388) [inline code_schemes/s02e01.json]: Would it make sense to call this LEAP S02E01?

**Review by as2388 — APPROVED** [review]

**Review by DanielMwendwa — APPROVED** [review]

File: code_schemes/s02e01.json (hunk 2 above)
Comment (IsaackMwenda) [reply code_schemes/s02e01.json]: Yup that would be useful. Updated

File: .idea/Project-WUSC-LEAP-II.iml (hunk 1 above)
Comment (IsaackMwenda) [reply .idea/Project-WUSC-LEAP-II.iml:1]: Thanks for catching this. Removed.

**Pull request merged**